content/micropython-course/course/01.installation/installation-tools.md

```diff
@@ -16,8 +16,8 @@ In this chapter we will go over how to install the prerequisite software: the **
 
 The software you will have installed are:
 
-- [Arduino Lab for MicroPython](https://labs.arduino.cc/en/labs/micropython)
-- The [Arduino MicroPython Installer](https://labs.arduino.cc/en/labs/micropython-installer)
+- Arduino Lab for MicroPython
```

Comment:
I would not remove the links here. I know they are listed further down, but I don't think it hurts having them here as well (some people might just want a quick access to the download link).

Maybe we can instead create a section like this at the top:
```md
## Software Requirements
In this chapter, we will be using the following software:
- [Installer]()
- [Editor]()
```

Reply:
The idea came after Giulio Tamberi went through the course and got confused and jumbled by exactly this, so it can definitely hurt the experience.
The crux is that seeing the link will tempt even the people who don't just want a quick link to leave the microPython 101 page for something else, and fall out of the courses linear path there. 

I do see your point about someone who might just want to find quick access to links though. 

I think we can find a middle ground. :)